native/go/digests

```diff
@@ -1,3 +1,3 @@
-go1.16.4.src.tar.gz SHA1 b1d5f9a63f0cda6580f6a31419af78dc0723536a
-go1.16.4.src.tar.gz SHA256 ae4f6b6e2a1677d31817984655a762074b5356da50fb58722b99104870d43503
-go1.16.4.src.tar.gz MD5 5884a4cb49067b9bcf6e69a1ef44259c
+go1.16.5.src.tar.gz SHA1 
+go1.16.5.src.tar.gz SHA256 
+go1.16.5.src.tar.gz MD5 
```

Comment:
use `make clean && make digests` to create valid digests file.

Reply:
Fixed in last commit.